.github/workflows/release.yaml

```diff
@@ -78,11 +78,10 @@ jobs:
                 # Release notes file
                 export RN_FILE="$(dirname $changelog)/RELEASE_NOTES.md"
                 export RN_CONTENT="$(awk '/^## Next release/{flag=1;next}/^## /{flag=0}flag' $changelog | awk 'NF {p=1} p')"
+                export CHANGELOG=$changelog
           
                 # ArtifactHub annotation in Chart.yaml file
-                # artifacthub expects content of changes annotation to be yaml escaped string 
-                # currently, it's not possible to properly escape it with shell and requires to write script with python/go/other pl
-                # yq -M -i eval '.annotations["artifacthub.io/changes"] = strenv(RN_CONTENT)' $CHART_FILE
+                yq -i '.annotations["artifacthub.io/changes"] = (load_str(strenv(CHANGELOG)) | capture("## Next release[\s]*(?P<changes>[\s\S]*?)[\s]*##") | "\(.changes)\n")' $CHART_FILE
```

Comment:
@AndrewChubatiuk , `.changes` can't seem to be templated correctly, see https://github.com/VictoriaMetrics/helm-charts/commit/521151c28c66957e0577b16cfc48e67415e9fdce, could you do another fix?

Reply:
reproduces this behaviour with old YQ version
created a [PR](https://github.com/VictoriaMetrics/helm-charts/pull/1119) with upgrades

Reply:
@Haleygo reproduces this behaviour with old YQ version
created a [PR](https://github.com/VictoriaMetrics/helm-charts/pull/1119) with upgrades

Reply:
I don't think that load_str is a good way to go, it tries to evaluate given line as yaml object. But do not perform escaping for it.